src/esbmc/bmc.cpp

```diff
@@ -175,10 +195,13 @@ smt_convt::resultt bmct::run_decision_procedure(
 
   std::stringstream ss;
   ss << "Solving with solver " << smt_conv->solver_text();
+  std::thread t1(printAliveMessage, msg);
   msg.status(ss.str());
 
   fine_timet sat_start = current_time();
   smt_convt::resultt dec_result = smt_conv->dec_solve();
+  bmc_interrupted = true;
+  t1.join();
```

Comment:
I don't think you should join... you will block the main thread for how much sleep there is left. Just detach it

Reply:
done